src/WebJobs.Extensions.EasyTables/Bindings/EasyTableOutputBindingProvider.cs

```diff
@@ -0,0 +1,101 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System;
+using System.Reflection;
+using System.Threading.Tasks;
+using Microsoft.Azure.WebJobs.Host.Bindings;
+using Microsoft.Azure.WebJobs.ServiceBus;
+
+namespace Microsoft.Azure.WebJobs.Extensions.EasyTables
+{
+    /// <summary>
+    /// Provides an <see cref="IBinding"/> for valid output parameters decorated with
+    /// an <see cref="EasyTableAttribute"/>.
+    /// </summary>
+    /// <remarks>
+    /// The method parameter type can be one of the following:
+    /// <list type="bullet">
+    /// <item><description><see cref="ICollector{T}"/>, where T is either <see cref="JObject"/> or any type with a public string Id property.</description></item>
```

Comment:
Are all these types you list correct for an out binding? I didin't think you could do "out ICollector<Foo>"?